tests/test_config.py

```diff
@@ -0,0 +1,261 @@
+from pathlib import Path
+
+import pytest
+
+import ssort._config as config
+
+
+def test_default_skip_defined():
+    assert hasattr(config, "DEFAULT_SKIP")
+
+
+class TestIterValidPythonFiles:
```

Comment:
For consitency, would it be possible to rewrite these tests using the pytest `test_` function style..